#ifndef MAINWINDOW_H
#define MAINWINDOW_H

#include <QMainWindow>

namespace Ui {
class MainWindow;
}

class MainWindow : public QMainWindow
{
    Q_OBJECT
    
public:
    explicit MainWindow(QWidget *parent = 0);
    ~MainWindow();
    
public slots:
    void ShowMainForm();
    void HideMainForm();

private slots:
    void on_bGames_clicked();

    void on_bJoin_clicked();

signals:
    void initializeChat();

private:
    Ui::MainWindow *ui;
};

#endif // MAINWINDOW_H
